### ISSUE TITLE
(0018,1114) EstimatedRadiographicMagnificationFactor is not used when performing measurements

### ISSUE THREAD

**Issue by alexturcea** [nroduit/Weasis#465]
### Describe the bug. What happened?

When performing measurements, the `(0018,1164) ImagerPixelSpacing` values should be corrected based on `(0018,1114) EstimatedRadiographicMagnificationFactor` (if available), or based on `(0018,1110) DistanceSourceToDetector` and `(0018,1111) DistanceSourceToPatient` (if both are available):

`EstimatedRadiographicMagnificationFactor = DistanceSourceToDetector / DistanceSourceToPatient`
`EffectivePixelSpacing = ImagerPixelSpacing / EstimatedRadiographicMagnificationFactor`

### What version of Weasis are you running?

4.2.0

### On which system the problem occurs?

Windows

**Comment by nroduit** [comment]
Weasis displays the different calibration methods according to IHE and DICOM recommendations, see the validation by running 'weasis://%24dicom%3Aget+-w+%22https%3A%2F%2Fnroduit.github.io%2Fdemo-archive%2Fdemo%2Fpixel-spacing.xml%22'

Using EstimatedRadiographicMagnificationFactor doesn't really guarantee accurate calibration.

![image](https://github.com/nroduit/Weasis/assets/993975/1e8e020c-540c-4845-b3cf-58a865638280)

**Comment by nroduit** [comment]
I've made [some modifications ](https://github.com/nroduit/Weasis/commit/951204a8ae34b37bf70af43f6e7d0c311f037ca7)to support the use of the magnification factor and applied it by default only for mammography according to the IHE recommendation of its profile.

However, following these modifications, it would be easy to add a preference to use the magnification factor instead of the detector. This could be added in a future release.

In short, as I understand it, the magnification factor need not be used except for mammography. What's important is to display the calibration method "At detector", "Magnified"...

**Comment by alexturcea** [comment]
Cool! I am not familiar with the IHE recommendations, but the magnification factor should also be used for x-ray angiography and fluoroscopy.

**Comment by nroduit** [comment]
It has been set for CR, DX, IO, MG, PX, RF, and XA.
